ksql-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java

```diff
@@ -455,6 +455,10 @@ private void waitForWarmStateStore(
   ) {
     // Special handling for pull queries is required, as they depend on materialized state stores
     // being warmed up.  Initial requests may return no rows.
+    if (querySql.contains("EMIT CHANGES")) {
```

Comment:
super nit: can we add a space between this line and the comment above? I was thrown for a bit thinking they were related.